llvm/lib/CodeGen/SelectionDAG/DAGCombiner.cpp

```diff
@@ -11516,9 +11516,18 @@ static SDValue combineMinNumMaxNumImpl(const SDLoc &DL, EVT VT, SDValue LHS,
   case ISD::SETLE:
   case ISD::SETULT:
   case ISD::SETULE: {
-    // Since it's known never nan to get here already, either fminnum or
-    // fminnum_ieee are OK. Try the ieee version first, since it's fminnum is
-    // expanded in terms of it.
+    // Since it's known never nan to get here already, either fminimumnum,
+    // fminimum, fminnum, or fminnum_ieee are OK. Try the ieee version first,
+    // since it's fminnum is expanded in terms of it.
```

Comment:
Can you explain in the comment why IEEE2019Num and IEEE2019 are preferred in that order?

Reply:
In some architectures FMINNUM is not defined well.
We are working on improve FMINNUM and finally remove FMINNUM_IEEE.
Currently, on some architectures that defines FMINNUM_IEEE , like AArch64/MipsR6, FMINNUM_IEEE is a better choice.